docs/API_REFERENCE.md

```
@@ -0,0 +1,861 @@
+    "classification": "chaotic",
+    "stability_score": 0.45,
+    "recommendations": [
+      "Refactorizar para reducir complejidad",
+      "Simplificar flujo de control",
+      "Considerar modularización"
+    ]
+  }
+  ```
+
+## ⚠️ Códigos de Error MCP
+## ⚠️ Códigos de Error MCP
```

markdownlint-cli2 (0.20.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)